frontend/app/player/components/FullScreenButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
Better to plase them separately, in `player-ui` folder, maybe, because `player`-core will be a separated library soon.